src/raft.rs

```diff
@@ -676,9 +680,20 @@ impl<T: Storage> Raft<T> {
         self.send(m);
     }
 
+    // bcast performs lazy broadcast strategy, no message will be broadcast until
+    // bcast_append is called.
+    pub fn bcast(&mut self) {
```

Comment:
I have mixed feelings about the name of this function.

When I see

```rust
let x =...
...
self.bcast();
...
```

I kind of assume it sends at the point of the `bcast` command. (I realize your comment makes this very obvious, but reading some of the other code I found myself a bit confused.) Maybe something like `pending_bcast` or something would be a more appropriate name?

Reply:
Yes, pending_bcast (or bcast_pending?) would be better, I will change it